pipeline_dp/pipeline_operations.py

```diff
@@ -78,8 +78,8 @@ def count_per_element(self, col, stage_name: str):
 class LocalPipelineOperations(PipelineOperations):
   """Local Pipeline adapter."""
   
-  def map(self, col, fn, stage_name: str):
-    pass
+  def map(self, col, fn, stage_name: str = None):
+    return map(fn, col)
```

Comment:
Here the order is reversed: `fn` is passed before `col`. That's how `map` works in most languages, so current order of the arguments in `PipelineOperations.map` may be confusing to people. Can we reverse it or should `col` stay as the first argument? Same thing with filter: function is passed before iterable to Python's filter.

Reply:
That's a good question! I think this is a common thing that different things are different in different languages/frameworks :)

For example in Spark and Beam collection is before fn in statement:

Beam:  col | beam.Map(fn)
Spark: col.Map(fn)

Another thing is that for all PipelineOperations method col is always an argument, fn isn't (eg. in Keys()), so it's more uniform to have col always as the 1st argument.